inclusion_connect/templates/403.html

```diff
@@ -0,0 +1,10 @@
+{% extends "layout/left_content.html" %}
+{% block content %}
+    {% if exception %}
+        <div class="alert alert-warning">
+            <p class="mb-0">{{ exception }}</p>
+        </div>
+    {% else %}
+        <h2 class="h1">Accès refusé</h2>
```

Comment:
C’est le premier heading du doc.
```suggestion
        <h1>Accès refusé</h1>
```